submissions/f3d0t/html_movie_seat_booking/index.html

```diff
@@ -0,0 +1,194 @@
+<!DOCTYPE html>
+<html lang="en" itemscope itemtype="http://schema.org/WebPage">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Fight Club | Movie ticket booking</title>
+		<meta name="description" content="Here you can book Fight Club movie tickets" />
+		<link rel="stylesheet" type="text/css" href="style.css" />
+		<link rel="preconnect" href="https://fonts.gstatic.com" />
+		<link href="https://fonts.googleapis.com/css2?family=Hind:wght@400;600&display=swap" rel="stylesheet" />
+	</head>
+
+	<body>
+		<header class="header">
+			<a class="header_logo" href="#"></a>
+			<nav>
+				<ul class="header_menu">
+					<li class="header_menu__item"><a href="#">Main</a></li>
+					<li class="header_menu__item"><a href="#">Ongoing</a></li>
+					<li class="header_menu__item header_menu__item--active"><a href="#">Booking</a></li>
+					<li class="header_menu__item"><a href="#">Bestsellers</a></li>
+					<li class="header_menu__item"><a href="#">Film rating</a></li>
+				</ul>
+			</nav>
+		</header>
+		<main class="main" id="main">
+			<nav>
+				<ul class="main_breadcrumbs" itemscope itemtype="https://schema.org/BreadcrumbList">
+					<li class="main_breadcrumbs__item" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem">
+						<a
+							itemscope
+							itemtype="https://schema.org/WebPage"
+							itemprop="item"
+							itemid="https://example.com/cinema/"
+							href="https://example.com/cinema/"
+						>
+							<span itemprop="name">Cinema</span></a
+						>
+						<meta itemprop="position" content="1" />
+					</li>
+					<li>›</li>
+					<li class="main_breadcrumbs__item" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem">
+						<a
+							itemscope
+							itemtype="https://schema.org/WebPage"
+							itemprop="item"
+							itemid="https://example.com/cinema/booking"
+							href="https://example.com/books/sciencefiction"
+						>
+							<span itemprop="name">Booking</span></a
+						>
+						<meta itemprop="position" content="2" />
+					</li>
+					<li>›</li>
+					<li class="main_breadcrumbs__item" aria-current="page" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem">
+						<span itemprop="name">Fight Club</span>
+						<meta itemprop="position" content="3" />
+					</li>
+				</ul>
+			</nav>
+			<article class="movie">
+				<div class="movie__info">
+					<h1 class="movie__title">"Fight club" movie tickets booking</h1>
+					<h2 class="movie__time">Session time & date: <time datetime="2021-02-14 13:45">14.02.21 / 13:45</time></h2>
```

Comment:
What the difference between this line and 66, that you choose heading `h2` there?

Reply:
because this heading is more informative, and It will be unique